pkg/subctl/benchmark/throughput.go

```diff
@@ -0,0 +1,62 @@
+package benchmark
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartThroughputTests() {
+	gomega.RegisterFailHandler(func(message string, callerSkip ...int) { panic(message) })
+	f := framework.NewBareFramework("throughput")
+	framework.ValidateFlags(framework.TestContext)
+	framework.BeforeSuite()
+
+	f.BeforeEach()
+	framework.By("Performing throughput tests from Gateway pod to Gateway pod")
```

Comment:
Even though the tests are successfully executed, IMHO the output is not clear enough for the user. Please take a close look at the output and see how we can present it in a more friendly manner. Thanks.